### ISSUE TITLE
Curious if script extender is back up and working for the most part?

### ISSUE THREAD

**Issue by HnyxLmn** [Norbyte/bg3se#331]
Just curious if anyone is able to play the game with the newest hot fixes and patch and has had minimal/no issues with script extender. I haven't updated to the newest hot fixes yet because I'm still a little worried it will break script extender and other mods. I've seen some people having issues the past week but it looks like Norbyte has been updating things. Not complaining, just curious if I should wait to update to the hot fixes yet or wait a bit longer!

**Comment by Vettic** [comment]
I've been able to play hotfix 20 last night, only issue I'm having is sometimes the game crashes when reloading a save, not the worst bug, I am using FocusBG's out of date mods that they've closed on nexus.

**Comment by Tarragon-L** [comment]
I get a warning about a version mismatch when I start the game since hotfix 20, but bg3se and all my mods work with no problems anyway.

**Comment by Vettic** [comment]
the launcher always drops a file mismatch warning when it's updated, I doubt it's a concern.

**Comment by HnyxLmn** [comment]
What does a version mismatch/ file mismatch mean exactly?

**Comment by JosephBelmont** [comment]
In my case, the game CTD as soon as the game loads to 100%, right before the main menu appears. I really don't know what's causing it.

**Comment by SheeraDarkphoenix** [comment]
> Just curious if anyone is able to play the game with the newest hot fixes and patch and has had minimal/no issues with script extender. I haven't updated to the newest hot fixes yet because I'm still a little worried it will break script extender and other mods. I've seen some people having issues the past week but it looks like Norbyte has been updating things. Not complaining, just curious if I should wait to update to the hot fixes yet or wait a bit longer!

I use both Vortex and BG3MM, Vortex for getting the mods and updating them. BG3MM for everything else. I created different mod orders to see if they would load into the game, along with the SE. I made one that started out with mods that didn't use the SE to see if the mods would load up into the game, as well as the SE. Once I saw the notification that said that the SE loaded up, I would exit out of the game, and slowly add in the mods that used the SE to test if they all loaded in or not. Once you have a really good working order, save the order as a test order (I label mine Test #). That way when the game updates, and you aren't sure what works or not, you have your test lists for just in case.
